packages/chain-space/src/ChainSpace.chain.ts

```diff
@@ -532,6 +532,67 @@ function dispatchDelegateAuthorizationTx(
   }
 }
 
+/**
+ * Prepares a delegate authorization extrinsic.
+ *
+ * @remarks
+ * Creates an extrinsic for delegating authorization based on the provided parameters.
+ *
+ * @param permission - The type of permission being granted.
+ * @param spaceId - The identifier of the space to which the delegate authorization is being added.
+ * @param delegateId - The decentralized identifier (DID) of the delegate receiving the authorization.
+ * @param authId - The identifier of the specific authorization transaction being constructed.
+ * @returns A promise resolving to the prepared extrinsic.
+ * @throws {SDKErrors.CordQueryError} - Thrown on error during preparation.
+ */
+export async function prepareDelegateAuthorizationExtrinsic(
```

Comment:
This function also requires `did.authorizeTx()` to be implemented for `prepare...`.

Refer `dispatchUpdateTxCapacityToChain()`.
The idea of prepare and dispatch is to separate `authorising`from did and `signing` from account.